docs/developer-onboarding.md

```diff
@@ -4,6 +4,16 @@
 
 This document describes the on-boardng steps for new Developers when they join the team.
 
+## You can request access to the following by asking in the #digital-tools-support slack channel:
+
+- [DfE-Digital](https://github.com/DFE-Digital) GitHub group access. Once you're added to the BAT (Becoming a Teacher) team you will have access to all repos. Ours are [Find](https://github.com/DFE-Digital/find-teacher-training) and [Apply](https://github.com/DFE-Digital/apply-for-teacher-training)
+
+
+- [Logit access](https://dashboard.logit.io/a/eeeb8311-79d8-49ab-9410-9b6d76b26f72) (Stack: Becoming a teacher - PaaS)
+
+
+- [Sentry access](https://sentry.io/auth/login/dfe-teacher-services/)
+
```

Comment:
Im trying to think of other tools we might need like Skylight

Reply:
Hmm, yeah do you have a link or some docs? 🤣 

Maybe google dev console too, what do you think?